File: app/src/main/java/info/hannes/cvscanner/sample/ScannerApplication.java
package info.hannes.cvscanner.sample;

import android.app.Application;

import info.hannes.timber.DebugTree;
import timber.log.Timber;

public class ScannerApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();
        Timber.plant(new DebugTree());
    }
}

File: app/src/main/java/info/hannes/cvscanner/sample/ScannerApplication.kt
package info.hannes.cvscanner.sample

import android.app.Application
import info.hannes.timber.DebugFormatTree
import timber.log.Timber

class ScannerApplication : Application() {
    override fun onCreate() {
        super.onCreate()
        Timber.plant(DebugFormatTree())
    }
}